internal/datastore/memdb/readwrite.go

```diff
@@ -163,7 +163,7 @@ func (rwt *memdbReadWriteTx) WriteNamespaces(newConfigs ...*core.NamespaceDefini
 	return nil
 }
 
-func (rwt *memdbReadWriteTx) DeleteNamespace(nsName string) error {
+func (rwt *memdbReadWriteTx) DeleteNamespaces(nsNames ...string) error {
 	rwt.lockOrPanic()
```

Comment:
we don't do tracing in memdb. I noticed this as I tested our tracing stack, and it would be nice to have it in place since it allows to very quickly test the tracing stack locally without spinning up a persistent datastore.

Let's open an issue for this?

Reply:
Fixes in #953